### ISSUE TITLE
Bug when swiping up from widget to access app drawer 

### ISSUE THREAD

**Issue by scoobertdoob** [amirzaidi/Launcher3#143]
When swiping upwards to access the drawer, if the swipe begins on a widget, the drawer fails to fully extend and instead the widget is highlighted for movement.

The subsequent touch places the widget at the given location.

**Comment by scoobertdoob** [comment]
To add:

This behaviour only occurs if the upwards swipe is a little slow. That is, if the initial tap is held for a even the tiniest amount of time before swiping up. This results in the drawer opening partially with the widget highlighted above/through it and other glitchy behaviour.

A rapid upwards swipe with no lingering on the initial tap results in typical behaviour.

**Comment by scoobertdoob** [comment]
http://imgur.com/a/4GaBe

Screenshot. Widget in question is Apple music. Reproducible with other widgets as well.

**Comment by amirzaidi** [comment]
I cannot reproduce this at all, can you give me more information about your setup? Which Android version is this?

**Comment by scoobertdoob** [comment]
Nexus 6P
Android 8.0.0
August security patch
Build Number: OPR6.170623.013

I suspected itd be tricky to reproduce. Try placing a widget just above the dock then when swiping up for the drawer, do it slowly so as to let the initial contact linger on the widget for a millisecond.

**Comment by scoobertdoob** [comment]
Alright, I tried doing it again a couple times and I can't seem to do it with any other widgets anymore. Only with the Apple music widget.
I guess I could've accidentally been hitting the touch target for the apple widget? I swear I did it with the keep widget once, but I can't reproduce it.

If it's exclusive to the Apple music widget I wouldn't worry about it. Apple  music on Android is a buggy piece of shit.
